### PR TITLE
update 6

### LINKED ISSUE #62: Update 6
- [x] Report Users and Posts UI (LiveView)
- [x] Approved Flag (Like Twitter (X) Blue Tick)
- [x] Translate to Russian (internationalized application via Phoenix gettext)
- [ ] Fix uploading Images on Posts, Profile and Cover 
- [ ] Post Some gif & emoji

### PR DESCRIPTION
**Pull request by 75pollet**
Fixes #62 
- [x] Add footer to all pages
- [x] Report Post
- [x] Report User 
- [x] Show the verified icon for verified users
- [x] Translate to Russian
- [x] Allow verifying and unverifying users

### CODE REVIEW

File: lib/mazaryn_web/templates/layout/app.html.heex [hunk 1]
```diff
@@ -2,4 +2,6 @@
   <p class="alert alert-info" role="alert"><%= get_flash(@conn, :info) %></p>
   <p class="alert alert-danger" role="alert"><%= get_flash(@conn, :error) %></p>
   <%= @inner_content %>
+
+  <%= render("_footer.html", assigns) %>
```
Comment (amos-kibet) [inline lib/mazaryn_web/templates/layout/app.html.heex:6]: @75pollet what's the reason for this change?

I'm working on #78 and I was applying some styles on the footer depending on whether the user is on the homepage or videos pages, and I was using `conn` to determine that. With this change, I don't have access to `conn` anymore.

Comment (ioavv) [reply lib/mazaryn_web/templates/layout/app.html.heex:6]: @75pollet He is waiting you to resolve it
